picard/tagger.py

```diff
@@ -510,82 +532,57 @@ def add_files(self, filenames, target=None):
         if new_files:
             log.debug("Adding files %r", new_files)
             new_files.sort(key=lambda x: x.filename)
+            self.window.set_sorting(False)
             if target is None or target is self.unclustered_files:
-                self.unclustered_files.add_files(new_files)
                 target = None
             for file in new_files:
                 file.load(partial(self._file_loaded, target=target))
+            self._pending_files_count += len(new_files)
```

Comment:
Should'nt `self._pending_files_count` set before the previous loop? Or even incremented in the loop:

```python
for file in new_files:
    self._pending_files_count += 1
    file.load(partial(self._file_loaded, target=target))
```

Reply:
Yup. Fixing.